src/main.rs

```diff
@@ -397,7 +398,7 @@ impl Drop for ClientStatus {
 }
 
 impl EventSender for ClientStatus {
-    fn send(&self, event: iroh_bytes::provider::Event) -> futures::prelude::future::BoxFuture<()> {
+    fn send(&self, event: iroh_bytes::provider::Event) -> Boxed<()> {
```

Comment:
You know why they did not copy BoxFuture but removed the lifetime?

Reply:
probably because in 95% of all use cases it is always `'static` anyways